### ISSUE TITLE
same recovery word repeated as part of 12-word seed

### ISSUE THREAD

**Issue by go140point6** [casinocoin/brm-issues#4]
This was noted by Hypto as he set up the BRM, but I had this happen with the linux version of the desktop wallet (5.0.7) as well.  One of my seed words repeated itself, #9 and #10 were the same.  Daniel seemed to think that probably shouldn't happen, although I have recovered successfully with those 12 words without issue into the mobile wallet several times.  I thought it weird, but figured whatever.

Maybe Hypto will comment with his specifics since it happened during his BRM setup.

**Comment by justfordaniel** [comment]
yes i had this in the brm and ia word was repeated on 9 and 12 if that makes any difference i dont know

**Comment by AKhashi** [comment]
I've got some strange behavior in BRM: I run KYC and got Operators screen with "pending KYC" message or something like this. After a w
hile I restarted the app and Operators screen is completely black now and rest of the screens are grey out
![brm_empty_ops](https://user-images.githubusercontent.com/59283454/75440064-b1551900-5963-11ea-99fa-cb9a848410d4.jpeg)
![brm_grey_trans](https://user-images.githubusercontent.com/59283454/75440083-bf0a9e80-5963-11ea-9534-0e8668e574c1.jpeg)

**Comment by ajochems** [comment]
Not a real problem but might be confusing. We will change this in a future version.
